nox/command.py

```diff
@@ -70,7 +75,7 @@ def run(
     paths: Optional[List[str]] = None,
     success_codes: Optional[Iterable[int]] = None,
     log: bool = True,
-    external: bool = False,
+    external: Union[Literal["error"], bool] = False,
```

Comment:
This was the confusing type that I noticed in the code that prompted this PR. The old type was incorrect, and should (and now is) have been caught by mypy.